tools/scoring/tests/parse_config_test.py

```diff
@@ -141,7 +141,7 @@ class _MockDimensionSimple(NamedTuple):
       def evaluate(self):
         return f'called with {self.translations}'
 
-    results = parse_config.ParseConfig.aggregate_results(
+    results = parse_config.ParseConfig._aggregate_results(  # pylint: disable=protected-access
```

Comment:
same here, dont access private methods from outside the class. Make them public

Reply:
Same question. Appreciate the consistency. :)